packages/sqflite/analysis_options.yaml

```diff
@@ -1,101 +0,0 @@
-include: package:flutter_lints/flutter.yaml
```

Comment:
Frontend package maintains this file. Is there a reason why it must be deleted?
https://github.com/tekartik/sqflite/blob/master/sqflite/analysis_options.yaml

Reply:
The file is for the [Dart code](https://github.com/tekartik/sqflite/tree/master/sqflite/lib) included in the package. There's no Dart code in our package (except `sqflite_tizen.dart`).